pgtype/bytea.go

```diff
@@ -154,3 +154,25 @@ func (src *Bytea) Value() (driver.Value, error) {
 		return nil, errUndefined
 	}
 }
+
+func (src *Bytea) MarshalJSON() ([]byte, error) {
+	switch src.Status {
+	case Present:
+		return src.Bytes, nil
```

Comment:
Raw bytes don't necessarily have a valid JSON representation, so I'm not sure if it makes sense to implement JSON marshaling.